components/ContractTokens.vue

```diff
@@ -28,18 +28,18 @@
 
     <v-card-actions class="justify-between">
       <MintTokens
-        v-if="contractState === 0"
+        v-if="contractState === 0 && !pastExpiry"
         :contract-details="contractDetails"
         :collateral-tokens="collateralTokens"
       />
       <RedeemTokens
-        v-if="contractState === 0"
+        v-if="contractState === 0 && !pastExpiry"
         :contract-details="contractDetails"
         :collateral-tokens="collateralTokens"
       />
       <!-- TODO: Later (when done with testing) we will want to show this only if the current date is past the expiry date  -->
       <ExpireContract
-        v-if="contractState === 0"
+        v-if="contractState === 0 && pastExpiry"
```

Comment:
we will probably want to make this a getter. No point in delaying the merge though.